docs/source/model_doc/mbart.rst

```diff
@@ -76,6 +68,87 @@ the sequences for sequence-to-sequence fine-tuning.
     assert translation == "Şeful ONU declară că nu există o soluţie militară în Siria"
 
 
+Overview of MBart-50
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MBart-50 was introduced in the `Multilingual Translation with Extensible Multilingual Pretraining and Finetuning
+<https://arxiv.org/abs/2008.00401>` paper by Yuqing Tang, Chau Tran, Xian Li, Peng-Jen Chen, Naman Goyal, Vishrav
+Chaudhary, Jiatao Gu, Angela Fan. mBART-50 is created using the original `mbart-large-cc25` checkpoint by extendeding
+its embedding layers with randomly initialized vectors for an extra set of 25 language tokens and then pre-trained on
+50 languages.
+
+According to the abstract
+
+*Multilingual translation models can be created through multilingual finetuning. Instead of finetuning on one
+direction, a pretrained model is finetuned on many directions at the same time. It demonstrates that pretrained models
+can be extended to incorporate additional languages without loss of performance. Multilingual finetuning improves on
+average 1 BLEU over the strongest baselines (being either multilingual from scratch or bilingual finetuning) while
+improving 9.3 BLEU on average over bilingual baselines from scratch.*
+
+
+Training of mBART-50
+_______________________________________________________________________________________________________________________
+
+The text format for mBART-50 is slightly different from mBART. For mBART-50 the language id token is used as a prefix
```

Comment:
bit of a nit, but we have both `Mbart-50` here and `mBART-50` -> maybe decide on one?

Reply:
I'd prefer `mBART-50`, since we also have `mBART` in the docs. A quick search & replace should make this change quite easy